dev/test-playbook.yaml

```diff
@@ -0,0 +1,8 @@
+- name: test-playbook
+  hosts: all
+  tasks:
+    - name: fuck me
```

Comment:
It would be nice to keep swearing to a minimum. We hope to make niployments a family friendly project